providers/aws/resources/aws.lr

```diff
@@ -1717,7 +1717,7 @@ aws.s3 @defaults("buckets") {
 }
 
 // Amazon S3 bucket
-private aws.s3.bucket @defaults("name location public") {
```

Comment:
Is there to do this without losing the location?

Reply:
unfortunately i dont think so..
:/

i aws returned the region on the list response this wouldnt be a problem, the extra api call is annoying

Reply:
Could we derive the location from the arn?

Reply:
unfortunately it's not in the arn for s3. i could also remove the location change stuff and just leave the rest of the optimization in

Reply:
chatted with tim, im gonna bring back the location by default part